src/components/ContainerPets/CardPet/styles.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import { fonts, fontSizes } from "../../../common/styles";
+
+const StyledCard = styled.figure`
+    display: flex;
+    margin: auto;
+    margin-bottom: 21px;
+    align-items: center;
+    width: 360px;
+    font-family: ${fonts.poppins};
```

Comment:
En vez de utilizar fonts.poppins , recomiendo que importes el theme y dentro de este tenemos fontPrimary 

```suggestion
    font-family: ${theme.fontPrimary};
```